library/src/main/java/com/pokegoapi/util/PokeNames.java

```diff
@@ -34,12 +34,6 @@
 	public static String getDisplayName(int pokedexNr, Locale locale) {
 		ResourceBundle names = ResourceBundle.getBundle("pokemon_names", locale);
 		String str = names.getString(String.valueOf(pokedexNr));
-		if (locale == Locale.FRENCH)
-			try {
-				return new String(str.getBytes("ISO-8859-1"), "UTF-8");
-			} catch (UnsupportedEncodingException e) {
-				e.printStackTrace();
-			}
 		return str;
```

Comment:
you can simplify the return now with return names.getString(String.valueOf(pokedexNr)); this doesn´t allocate another value that has to get cleaned from the gc.